src/legacy/core_plugins/kibana/public/visualize/embeddable/visualize_embeddable.ts

```diff
@@ -149,8 +149,6 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
 
     this.vis._setUiState(this.uiState);
 
-    this.appState = appState;
```

Comment:
:information_source: This line is just redundant to line #132 above, which already does the same thing.